TaxFoundation/country-landing-pages
# Cu 2q567h8 gatsby update

# Details
This PR updates Gatsby and all Gatsby plugins to their latest versions as well as adding strengths and weaknesses to the ITCI report widget. 

# Steps to test
1. run 'npm install'
2. run 'npm run develop'
3. go to 'localhost:8000' and click on a /itci/ link to check new strengths and weaknesses functionality 
4. click on one of each of the other links to ensure gatsby update did not break them. 

# Issue Addressed
[2kbwqq0](https://app.clickup.com/t/2kbwqq0)

# Screenshots
![Screen Shot 2022-08-15 at 2 39 39 PM](https://user-images.githubusercontent.com/29612767/184927606-c1892618-7e34-4892-a1bf-4195bf25741a.png)
